### DISCUSSION TITLE
Build Battle

### DISCUSSION THREAD

**Discussion by mr-breakfast** [MauritsWilke/HyAPI#20] [Reformatting Notice]
Notifying you that I'd like to reformat build battle

**Comment by MauritsWilke** [comment]
Okay thanks!!

Good luck with that :octocat:

**Reply by mr-breakfast** [reply]
thanks, will prob need a bit of help understanding

**Reply by MauritsWilke** [reply]
Feel free to shoot me any questions you might have! Always open for suggestions as well
